tests/integration/queries/test_workflow.py

```diff
@@ -210,3 +215,58 @@ def test_workflow_submission_auto_review(
     job = client.call(JobStatus(job.id))
     submission = client.call(GetSubmission(sub.id))
     assert submission.status == "COMPLETE" if force_complete else "PENDING_REVIEW"
+
+
+def _new_dataset_for_updating(client):
+    # new dataset
+    airline_csv = str(Path(__file__).parents[1]) + "/data/AirlineComplaints.csv"
```

Comment:
do any preexisting tests fail because of the csv column reordering?

Reply:
none that i can tell

Reply:
and here I've been using ".." / ".."